app/models/manageiq/providers/azure/cloud_manager/cloud_database.rb

```diff
@@ -43,8 +43,22 @@ def self.params_for_create(ems)
             }
           end,
         },
-        # TODO: Database Server dropdown is dependent on 'database type' and 'resource group'
-        # so it cannot be handled here, it will be handled in the UI repo
+        # TODO: Database Server dropdown should be dependent on 'database type' and 'resource group'
+        {
+          :component    => 'select',
+          :name         => 'server',
+          :id           => 'server',
+          :label        => _('Database Server Name'),
+          :includeEmpty => true,
+          :isRequired   => true,
+          :validate     => [{:type => 'required'}],
+          :options      => ems.cloud_database_servers.map do |srv|
+            {
+              :label => srv.name,
+              :value => srv.name,
```

Comment:
@nasark just want to confirm that it is the server name which is meant to be passed to the Azure API (and not e.g. the ems_ref)

Reply:
Yes, the server name is passed to the Azure API when creating a cloud database. For example:
```
> ManageIQ::Providers::Azure::CloudManager::CloudDatabase.raw_create_cloud_database(ems, {:database => "SQL", :name => "test", :resource_group => "test-group", :server => "miq-sql-server"})
```